### DISCUSSION TITLE
Support for AWS MQ and AWS S3

### DISCUSSION THREAD

**Discussion by sl4dy** [gravitational/teleport#47082] [Q&A]
Hi,
Can we use Teleport to manage access to AWS MQ and AWS S3? Thx!

**Comment by webvictim** [comment]
Yes - https://goteleport.com/docs/enroll-resources/application-access/cloud-apis/aws-console

**Reply by sl4dy** [reply]
Thx @webvictim . I understand that I can manage any AWS resource in general. But to access/connect to AWS MQ for Rabbit, I have to use IAM and sign the request. As stated in https://github.com/gravitational/teleport/issues/23435, signing does not work with Teleport in general.
I know I can use the workaround mentioned in the issue, but something more native would be welcome.
